docs/source/developers.md

```diff
@@ -17,11 +17,24 @@ To run all tests in docker containers (tests against many versions of python), r
 docker-compose up --build && docker-compose down
 ```
 
+## Making a Pull Request
+Feel free to fork this repo and submit a PR! We do trunk-based development so all PRs should be destined for the
+`main` branch. Reviews are required before merging and our automated tests must pass. Please see the following
+checklist for a basic set of requirements before we will merge a PR.
+
+### Checklist
```

Comment:
Maybe add this into a PR template with checkboxes for contributors?
https://github.com/IMAP-Science-Operations-Center/imap_processing/blob/dev/.github/pull_request_template.md

I do like how nice and contained this is.

Reply:
Good suggestion on the PR template. I didn't know about that.

I need another Approval since this changed the PR after the last approval. Then I think this is ready.